docs/references/config/config.toml.md

```diff
@@ -1256,32 +1256,41 @@ If the mempool is full, incoming transactions are dropped.
 
 The value `0` is undefined.
 
-### mempool.max_txs_bytes
-The maximum size of all transactions accepted in the mempool.
+### mempool.max_tx_bytes
+Maximum size in bytes of a single transaction accepted into the mempool.
 ```toml
-max_txs_bytes = 1073741824
+max_tx_bytes = 1048576
 ```
 
 | Value type          | integer |
 |:--------------------|:--------|
 | **Possible values** | &gt;= 0 |
 
-This is the raw, total transaction size. Given 1MB transactions and a 5MB maximum transaction size, mempool will only
-accept five transactions.
+Transactions bigger than the maximum configured size are rejected by mempool,
+this applies to both transactions submitted by clients via RPC endpoints, and
+transactions receveing from peers on the mempool protocol.
 
-The default value is 1 Gibibyte (2^30 bytes).
-
-### mempool.max_tx_bytes
-Maximum size of a single transaction accepted into the mempool.
+### mempool.max_txs_bytes
+The maximum size in bytes of all transactions stored in the mempool.
 ```toml
-max_tx_bytes = 1048576
+max_txs_bytes = 67108864
 ```
 
 | Value type          | integer |
 |:--------------------|:--------|
 | **Possible values** | &gt;= 0 |
 
-This is the maximum size of a transaction allowed to be accepted into the mempool.
+This is the raw, total transaction size. For example, given 1MB
+transactions and a 5MB maximum mempool byte size, the mempool will
+only accept five transactions.
+
+The maximum mempool byte size should be a factor of the network's maximum block size
+(which is a [consensus parameter](https://docs.cometbft.com/v1.0/spec/abci/abci++_app_requirements#blockparamsmaxbytes)).
+The rationale is to consider how many blocks have to be produced in order to
+drain all transactions stored in a full mempool.
```

Comment:
Should we add a blurb of text saying what happens if the mempool is "full" ? We added that to the max tx size, but probably would be important to also highlight the fact that if the mempool is considered full txs won't be accepted ?